File: cloudus-bugsmash/GCD and LCM - 2/c/GCD_LCM.c
#include <stdio.h>
#include <stdlib.h>

int gcd(int a, int b) {
    while (a != 0) {
        int temp = a;
        a = b;
        b = temp % b;
    }
    return b;
}

int lcm(int a, int b) {
    return abs(a * b) / gcd(a, b);
}

int main() {
    int num1, num2;

    printf("Enter the first number: ");
    scanf("%d", &num1);
    printf("Enter the second number: ");
    scanf("%d", &num2);

    int gcd_result = gcd(num1, num2);
    int lcm_result = lcm(num1, num2);

    printf("The GCD of %d and %d is %d\n", num1, num2, gcd_result);
    printf("The LCM of %d and %d is %d\n", num1, num2, lcm_result);

    return 0;
}

File: cloudus-bugsmash/GCD and LCM - 2/python/GCD_LCM.py
def gcd(a, b):
    while a:  
        b, a = a, b % a  
    return b  


def lcm(a, b):
    return abs(a * b) / gcd(a, b)  


num1 = input("Enter the first number: ")  
num2 = input("Enter the second number: ")


gcd_result = gcd(num1, num2)
lcm_result = lcm(num1, num2)


print("The GCD of " + num1 + " and " + num2 + " is " + str(gcd_result))  
print("The LCM of " + num1 + " and " + num2 + " is " + str(lcm_result))
